common/workunit/workunit.cpp

```diff
@@ -3004,6 +3005,7 @@ class CDaliWorkUnitFactory : public CWorkUnitFactory, implements IDaliClientShut
                 conn->reload();
             }
         }
+        waiter->unsubscribe();
```

Comment:
@jakesmith - I added this line back in - it is in all the others and was removed for 6.0
Do you think it should be here ?